src/main/java/net/mcreator/ui/views/editor/image/ImageMakerView.java

```diff
@@ -51,14 +53,16 @@
 import java.awt.event.MouseMotionListener;
 import java.io.File;
 import java.io.IOException;
+import java.util.Arrays;
+import java.util.Objects;
 import java.util.concurrent.ExecutorService;
 import java.util.concurrent.Executors;
 
 public class ImageMakerView extends ViewBase implements MouseListener, MouseMotionListener {
 
 	private static final Logger LOG = LogManager.getLogger("Image Maker View");
 
-	private String name = "Texture creator";
+	private String name;

```

Comment:
This should be set to `L10N.t("tab.image_maker")` for a safety fallback so name is never empty